<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GTFSDialog</class>
 <widget class="QDialog" name="GTFSDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>201</width>
          <height>50</height>
         </size>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Folder name</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditTrace">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>191</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxInputProj">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>191</width>
           <height>26</height>
          </rect>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>161</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Input projection format</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxOutputProj">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>191</width>
           <height>26</height>
          </rect>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>161</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Output projection format</string>
         </property>
        </widget>
        <zorder>label</zorder>
        <zorder>lineEditTrace</zorder>
        <zorder>comboBoxInputProj</zorder>
        <zorder>label_9</zorder>
        <zorder>comboBoxOutputProj</zorder>
        <zorder>label_10</zorder>
        <zorder>buttonBox</zorder>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
